### PR TITLE
Bump deps all

### PR DESCRIPTION
**Pull request by pradeepitm12**
# Changes
tektoncd/pipeline: * -> 0.32.0
tektoncd/triggers: * -> 0.17.1
tektoncd/hub: * -> latest
knative.dev/pkg: latest

Signed-off-by: Pradeep Kumar <pradkuma@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

### CODE REVIEW

**Comment by pradeepitm12** [comment]
Seems we are getting errors after updating to triggers v0.18.0 as the test builders are removed in v0.18.0 https://github.com/tektoncd/triggers/pull/1252
While testing it was observed that tkn build works fine with the update but the test fails and the mentioned pr could be the reason( we will need to update the tests)

For the release, lets go with triggers 0.17.1 and will try to update the triggers test in the upcoming pr.
cc: @vinamra28 @piyush-garg @vdemeester

**Comment by vinamra28** [comment]
> Seems we are getting errors after updating to triggers v0.18.0 as the test builders are removed in v0.18.0 [tektoncd/triggers#1252](https://github.com/tektoncd/triggers/pull/1252) While testing it was observed that tkn build works fine with the update but the test fails and the mentioned pr could be the reason( we will need to update the tests)
> 
> For the release, lets go with triggers 0.17.1 and will try to update the triggers test in the upcoming pr. cc: @vinamra28 @piyush-garg @vdemeester

@pradeepitm12 I guess we aren't using any test builders in the `cli` but yeah let's do the release with 0.17.1 and post that we can look into it

File: docs/cmd/tkn_hub_downgrade_task.md [hunk 1]
```diff
@@ -35,8 +35,7 @@ Downgrade a Task of name 'foo' to version '0.3':
 ### Options inherited from parent commands
 
 ```
-      --api-server string   Hub API Server URL (default 'https://api.hub.tekton.dev').
-                            URL can also be defined in a file '$HOME/.tekton/hub-config' with a variable 'HUB_API_SERVER'.
+      --api-server string   Hub API Server URL (default "https://api.hub.tekton.dev")
```
Comment (vinamra28) [inline docs/cmd/tkn_hub_downgrade_task.md:38]: /hold
@piyush-garg @PuneetPunamiya any idea when did it got changed :thinking:

Comment (vinamra28) [reply docs/cmd/tkn_hub_downgrade_task.md:38]: nvm I found the problem :sweat_smile:

File: go.mod [hunk 2]
```diff
@@ -9,23 +9,22 @@ require (
...
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/opencontainers/image-spec v1.0.2 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
-	github.com/tektoncd/hub/api v0.0.0-20210922050526-aa589697cb58
-	github.com/tektoncd/pipeline v0.28.0
-	github.com/tektoncd/plumbing v0.0.0-20210514044347-f8a9689d5bd5
-	github.com/tektoncd/triggers v0.16.0
+	github.com/tektoncd/hub/api v0.0.0-20210315114749-a667b06c02ac
```
Comment (vinamra28) [inline go.mod:24]: @pradeepitm12 can you please bump the latest version of hub? This commit belongs to 5th March, 2021

Comment (pradeepitm12) [reply go.mod:24]: copy paste from other pr. :stuck_out_tongue:

**Review by vdemeester — APPROVED** [review]
/meow

**Pull request merged**